src/TEC/Plugin.php

```diff
@@ -0,0 +1,392 @@
+	}
+
+	/**
+	 * Adjust the label for RSVP Attendees to reflect attendee type.
+	 *
+	 * @see https://docs.theeventscalendar.com/reference/hooks/tribe_tickets_register_attendee_post_type_args/
+	 *
+	 * @param array $args Post type arguments.
+	 *
+	 * @return array
+	 */
+	function rsvp_attendees_label( $args ) {
+		return $this->container->make( Post_Handler::class )->rsvp_attendees_label( $args );
+	}
+
+	/**
+	 * Adjust the label for Tribe Commerce Orders to reflect eCommerce provider.
+	 * Note: Tribe Commerce has been deprecated in favor of Tickets Commerce.
+	 *
+	 * @see https://docs.theeventscalendar.com/reference/hooks/tribe_tickets_register_order_post_type_args/
+	 *
+	 * @param array $args Post type arguments.
+	 *
+	 * @return array
+	 */
+	function tpp_orders_label( array $args ): array {
+		return $this->container->make( Post_Handler::class )->tpp_orders_label( $args );
+	}
+
+	/**
+	 * Adds '_EventOrigin' to the tracked keys.
+	 * Note: Updating a tracked key triggers the creation or update of the Custom Table entries.
+	 *
+	 * Allows filtering the list of meta keys that, when modified, should trigger an update to the custom tables’ data.
+	 *
+	 * @since   1.0.0
+	 *
+	 * @param array $tracked_keys Array of the tracked keys.
+	 *
+	 * @return array
+	 *
+	 * @see     https://docs.theeventscalendar.com/reference/hooks/tec_events_custom_tables_v1_tracked_meta_keys/
+	 *
+	 * @see     \TEC\Events\Custom_Tables\V1\Updates\Meta_Watcher::get_tracked_meta_keys()
+	 */
+	public function modify_tracked_meta_keys( array $tracked_keys ): array {
+		return $this->container->make( Post_Handler::class )->modify_tracked_meta_keys( $tracked_keys );
+	}
+
+	/**
+	 * Checks whether the plugin dependency manifest is satisfied or not.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return bool Whether the plugin dependency manifest is satisfied or not.
+	 */
+	protected function check_plugin_dependencies(): bool {
+		$this->register_plugin_dependencies();
+
+		return tribe_check_plugin( static::class );
+	}
+
+	/**
+	 * Registers the plugin and dependency manifest among those managed by Tribe Common.
+	 *
+	 * @since 1.0.0
+	 */
+	protected function register_plugin_dependencies(): void {
+		$plugin_register = new Plugin_Register();
+		$plugin_register->register_plugin();
+
+		$this->container->singleton( Plugin_Register::class, $plugin_register );
+		$this->container->singleton( 'extension.wpai_add_on', $plugin_register );
+	}
+
+	/**
+	 * Get this plugin's options prefix.
+	 *
+	 * Settings_Helper will append a trailing underscore before each option.
+	 *
+	 * @return string
+	 *
+	 * @see \TEC\Extensions\WpaiAddOn\Settings::set_options_prefix()
+	 *
+	 * TODO: Remove if not using settings
+	 */
+	private function get_options_prefix(): string {
+		return (string) str_replace( '-', '_', 'tec-labs-wpai-add-on' );
+	}
+
+	/**
+	 * Get Settings instance.
+	 *
+	 * @return Settings
+	 *
+	 * TODO: Remove if not using settings
```

Comment:
Looks like we're using settings here (and below), should we remove the TODOs?

Reply:
While we could use some settings, i.e. retain blank meta entries, we don't necessarily have to. These are more for developers / professionals who can use the filters.